### ISSUE TITLE
Make `swagger_dry_run` configuration by default `false`

### ISSUE THREAD

**Issue by SagareGanesh** [rswag/rswag#266]
As per documentation
we need to disable --dry-run option for Rspec > 3

Add to application.rb:
```
RSpec.configure do |config|
  config.swagger_dry_run = false
end
```

The Issue here is, In development or test environment its working correctly.
But In production environment its failing, as we are not loading Rspec in production env.

There is 2 solutions for it, 
1. load above configuration only in development environment. i.e
```
    if Rails.env.development?
      RSpec.configure do |config|
        config.swagger_dry_run = false
      end
    end
```
2. Set dry run false by default, and if user wants they can turn it true.

I have raised PR for solution 2.

**Comment by igas** [comment]
We had the same concerns, pls take a look at my attempt https://github.com/rswag/rswag/pull/274

**Comment by jtannas** [comment]
It looks like the underlying concern is resolved by #274 which has been released so I'm going to go ahead and close this now. Thank you for the contributions though :slightly_smiling_face:
